### DISCUSSION TITLE
Node export should be removed from @types/xast

### DISCUSSION THREAD

**Discussion by remcohaszing** [syntax-tree/unist#30] [General] [answered]
`@types/xast` re-exports the `unist.Node` type. However, `unist.Node` is unrelated to `xast`, except that `xast.Doctype` extends it.

I think this leads to confusion, causing for example https://github.com/syntax-tree/xastscript/issues/8.

It should probably be removed, but that would be a breaking. Should it be deprecated instead?

**Comment by ChristianMurphy** [comment, marked as answer]
@remcohaszing fair point, let's start with deprecating it, and we can remove it in the next major
